dependencies.props

```diff
@@ -10,17 +10,17 @@
   -->
   <PropertyGroup>
     <CoreFxCurrentRef>490015de35bfb73a258bb78ecd5241cc6e207908</CoreFxCurrentRef>
-    <CoreClrCurrentRef>490015de35bfb73a258bb78ecd5241cc6e207908</CoreClrCurrentRef>
-    <StandardCurrentRef>807c5b1aab825f6c4fde32b333bbe7d6ab59678e</StandardCurrentRef>
-    <BuildToolsCurrentRef>e5445e389a5f7a4058158e12aee08b40ffafa09c</BuildToolsCurrentRef>
+    <CoreClrCurrentRef>270a80f37f01e09111b7b12858f1d0424bf9983b</CoreClrCurrentRef>
+    <StandardCurrentRef>270a80f37f01e09111b7b12858f1d0424bf9983b</StandardCurrentRef>
+    <BuildToolsCurrentRef>270a80f37f01e09111b7b12858f1d0424bf9983b</BuildToolsCurrentRef>
   </PropertyGroup>
 
   <PropertyGroup>
     <MicrosoftNETCorePlatformsPackageVersion>2.2.0-preview1-26517-01</MicrosoftNETCorePlatformsPackageVersion>
     <MicrosoftNETCoreTargetsPackageVersion>2.0.0</MicrosoftNETCoreTargetsPackageVersion>
     <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.6.0-preview1-26517-01</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
-    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.2.0-preview1-26517-01</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
-    <NETStandardLibraryPackageVersion>2.0.3</NETStandardLibraryPackageVersion>
+    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.2.0-preview1-26522-04</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
+    <NETStandardLibraryPackageVersion>2.0.4-servicing-26522-02</NETStandardLibraryPackageVersion>
```

Comment:
@weshaggard @ericstj - is it expected that `dotnet/standard` is now producing a `2.0.4-servicing` version and our `2.2` master build is dependent on it?

Reply:
This came from my build of standard here - https://devdiv.visualstudio.com/DevDiv/Net%20CoreFX/_build/index?buildId=1710206&_a=summary

To test changes to implement Microbuild signing. I also bumped the versions in this PR - https://github.com/dotnet/standard/pull/728/files#diff-3509e8cf826f0a69fdac329923d825aaR6

Reply:
It is expected that we bumped the version 2.0.4 but I do think we want to keep corefx on the last stable version so we may want to pin the standard version or come up with some other way to get it back to last stable. It is OK for this to merge for now it won't matter until we try to ship 2.2.